third_party/blink/renderer/modules/indexeddb/idb_database.cc

```diff
@@ -114,8 +114,13 @@ IDBDatabase::IDBDatabase(
 }
 
 IDBDatabase::~IDBDatabase() {
-  if (!close_pending_ && backend_)
-    backend_->Close();
+  if (!close_pending_ && backend_) {
+    // Explicitly leak the database proxy, as we are likely in a GC, and
+    // closing will issue IPC messages that need to be recorded.
+    if (!recordreplay::IsRecordingOrReplaying("IDBDatabase::~IDBDatabase")) {
```

Comment:
This feature string needs to be one of the features named in api.cc in V8.  This makes sure we know the full set of features and can easily disable them.  It's generally better to use an existing feature for small changes like this so that we don't end up with a huge list of them.  In this case using the `leak-references` feature seems appropriate.